### DISCUSSION TITLE
google-photos/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#122] [General]
# google-photos/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/google-photos/

**Comment by xpil** [comment]
Używam Google Photos "niechcący", bo mi to kiedyś włączyli na koncie, nic mnie to nie kosztuje i faktycznie mam kopię zapasową swoich fotek, choć w nieoryginalnej rozdzielczości. Jednak moim głównym miejscem na składowanie wszelakich multimediów jest chmura Dropbox (2TB za €100 rocznie). Tam siedzą wszystkie oryginały. Dropboxa synchronizuję z paroma urządzeniami lokalnymi, a oprócz tego mam bieda-dysk usb 1TB, na który wykonuję kopię off-line raz na jakiś czas.
